src/games/strategy/engine/data/export/RulesAttachmentExporter.java

```diff
@@ -134,11 +135,9 @@ private String mTurnsHandler(final Field field, final IAttachment attachment) {
     try {
       oValue = field.get(attachment);
     } catch (final IllegalArgumentException e) {
-      // TODO Auto-generated catch block
-      e.printStackTrace();
+      ClientLogger.logError(e);
```

Comment:
Please create an error message string similar to:
`String failMsg = "Failed to get attachment: "+ attachment + ", from field: " + field;`
Then call client logger with the error mssage
`logError(failMsg, e)`

Otherwise if this fails we'll be searching for how to reproduce the problem. The extra information skips debugging completely.